xla/stream_executor/dnn.h

```diff
@@ -1706,6 +1710,29 @@ class DnnSupport {
       std::optional<dnn::TensorDescriptor> dscale_descriptor,
       std::optional<dnn::TensorDescriptor> dbias_descriptor);
 
+  virtual absl::StatusOr<bool> PrepareGraph(cudnn_frontend::graph::Graph&) {
+    return absl::UnimplementedError(
+        "DnnSupport::PrepareGraph is specific to CUDA.");
+  }
+
+  virtual absl::Status BuildGraph(cudnn_frontend::graph::Graph&,
+                                  const int64_t) {
+    return absl::UnimplementedError(
+        "DnnSupport::BuildGraph is specific to CUDA.");
+  }
+
+  virtual absl::Status DeserializeGraph(cudnn_frontend::graph::Graph&,
```

Comment:
`absl::StatusOr<std::unique_ptr<se::dnn::DnnGraph>> DeserializeGraph(string_view)` should be exposed from DnnSupport, and everything else can be a DnnGraph virtual method

Reply:
Done.